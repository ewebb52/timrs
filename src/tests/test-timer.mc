int main() 
{
    timer_init(1);
    start_timer(2000);
    timer_destroy(1);
}
