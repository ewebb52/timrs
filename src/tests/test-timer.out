Timer ran for 2000